src/Components/App/App.js

```diff
@@ -61,22 +61,24 @@ class App extends Component {
   }
 
   addTrack(track) {
-    const playlistTracks = this.state;
+    const { playlistTracks } = this.state;
+
     if (playlistTracks.find(savedTrack => savedTrack.id === track.id)) {
       return;
-    } else {
-      playlistTracks.push(track);
-      this.setState({ playlistTracks });
     }
+
```

Comment:
No need for an `else` here since you just return out of the `if`

Reply:
Also, this `if` means you can't have the same track more than once in your playlist … just FYI